### PR TITLE
fix bugs in xhr reqeust

### PR DESCRIPTION
**Pull request by mossa-Sammer**
relates #30

### CODE REVIEW

File: public/js/request.js [hunk 1]
```diff
@@ -5,10 +5,10 @@ const createXhr = (url, method, search, callback) => {
     if (xhr.readyState === 4 && xhr.status === 200) {
       console.log(2222222222);
       const data = xhr.responseText;
-      callback(data, xhr.status);
+      callback(data,xhr.status);
```
Comment (MohammadAlhalaq) [inline public/js/request.js]: Put space after data

File: public/js/request.js [hunk 2]
```diff
@@ -5,10 +5,10 @@ const createXhr = (url, method, search, callback) => {
...
     } else if (xhr.readyState === 4) {
       console.log(333333333)
-      callback(undefined, xhr.status);
+      callback(undefined,xhr.status);
```
Comment (MohammadAlhalaq) [inline public/js/request.js]: space

**Review by fadeomar — APPROVED** [review]

File: public/js/request.js (hunk 2 above)
Comment (mossa-Sammer) [reply public/js/request.js]: solved ^_^

File: public/js/request.js (hunk 1 above)
Comment (mossa-Sammer) [reply public/js/request.js]: Solved ^_^

**Review by fadeomar — APPROVED** [review]

**Review by mohqarmout — APPROVED** [review]

**Pull request merged**